react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react"
+import './Accordion.css'
+
+export default function Accordion({ topicsArray }) {
+  const [currentIndex, setCurrentIndex] = useState(null);
+
+  function HandleOnClick(index) {
+    index === currentIndex ? setCurrentIndex(null) : setCurrentIndex(index);
+  }
+
+  return (
+    <div className="container">
+      <Buttons data={topicsArray} current={currentIndex} onClick={HandleOnClick} />
+    </div>
+  );
+}
+
+function Buttons({ data, current, onClick }) {
+  const buttons = data.map((topic, index) => (
+    <Button
+      key={index}
+      text={topic.title}
+      onClick={() => onClick(index)}
+      isActive={current === index}
+      description ={topic.description}
+      />
+  ));
+
+  return (
```

Comment:
This component is pretty simple. I'm not sure it carries its weight enough to be worth making a separate component. I probably would have just inlined this up above in Accordion since it's so simple.